homework-6/script-1.js

```diff
@@ -0,0 +1,83 @@
+//Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+const getMinNumber = (num_1, num_2, num_3, num_4) => {
```

Comment:
It still works with strings
<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191561801-081df490-561d-495c-afe3-7e917ebfe1f4.png">